c2_components/src/mfx_c2_decoder_component.cpp

```diff
@@ -1341,8 +1341,8 @@ c2_status_t MfxC2DecoderComponent::UpdateC2Param(const mfxVideoParam* src, C2Par
         case kParamIndexPictureSize: {
             if (C2StreamPictureSizeInfo::output::PARAM_TYPE == index) {
                 MFX_DEBUG_TRACE("GetPictureSize");
-                m_size->width = src->mfx.FrameInfo.CropW;
-                m_size->height = src->mfx.FrameInfo.CropH;
+                m_size->width = src->mfx.FrameInfo.Width;
```

Comment:
Did we check 1080p video playback? Is the output 1920x1080 instead of 1920x1088?

Thanks,
Tianmi

Reply:
okay, let's wait for the cts results.

Thanks,
Tianmi